src/sbvr-api/odata-response.ts

```diff
@@ -158,9 +162,17 @@ export const process = async (
 	);
 
 	const odataIdField = sqlNameToODataName(table.idField);
+	const webResourceHandler = getWebresourceHandler();
 	for (const row of rows) {
 		for (const fieldName of processedFields) {
 			row[fieldName] = fetchProcessingFields[fieldName](row[fieldName]);
+			if (
+				webResourceHandler &&
+				isWebResourceField(fieldName, table) &&
+				row[fieldName]?.href != null
```

Comment:
Checking `row[fieldName]?.href != null` should be encapsuled in a webresource function as it is webResource specific knowledge